examples/sprite_animation/main.rs

```diff
@@ -0,0 +1,245 @@
+//! Demonstrates how to load and render sprites.
+//!
+//! Sprites are from <https://opengameart.org/content/bat-32x32>.
+
+use amethyst::{
+    animation::{
+        get_animation_set, AnimationBundle, AnimationCommand, AnimationSet, AnimationSetPrefab,
+        EndControl,
+    },
+    assets::{AssetStorage, JsonFormat, Loader, PrefabData, PrefabLoaderSystem, ProgressCounter},
+    config::Config,
+    core::transform::{Transform, TransformBundle},
+    derive::PrefabData,
+    ecs::{prelude::Entity, Read, ReadExpect, WriteStorage},
+    error::Error,
+    prelude::{Builder, World},
+    renderer::{
+        Camera, DisplayConfig, DrawFlat2D, Pipeline, Projection, RenderBundle, ScreenDimensions,
+        Sprite, SpriteRender, SpriteSheet, Stage, TextureFormat, TexturePrefab,
+    },
+    utils::application_root_dir,
+    Application, GameData, GameDataBuilder, SimpleState, SimpleTrans, StateData, Trans,
+};
+use amethyst_assets::PrefabLoader;
+use serde::{Deserialize, Serialize};
+
+/// Information about position of the sprite in the sprite sheet
+/// Positions originate in the top-left corner (bitmap image convention).
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+struct SpritePosition {
```

Comment:
could you make the `SpritePosition` and `SerializedSpriteSheet` public in the renderer crate and use those instead?

I don't think there's strong reason for them to be private; and thinking about it a bit more, `SpriteRenderPrefab` is worth putting into the renderer crate -- it's good 😄!

Reply:
Should I open a separate pull request for `SpriteRenderPrefab`?

Reply:
I think it's alright to be in this one